k8s/arroyo/values.yaml

```diff
@@ -13,9 +13,10 @@ controller:
   image:
     repository: ghcr.io/arroyosystems/arroyo
     pullPolicy: IfNotPresent
-    tag: "dev"
+    tag: "df_deltalake_docker"
```

Comment:
revert?

also—you can override this with a values.yaml file so it's not generally required to change the repo